packages/grid/backend/grid/db/init_db.py

```diff
@@ -16,28 +18,36 @@ def init_db(db: Session, signing_key: Optional[SigningKey] = None) -> None:
         if signing_key is None:
             signing_key = SigningKey.generate()
 
-        node.initial_setup(
-            signing_key=signing_key,
-            first_superuser_name="Jane Doe",
-            first_superuser_email=settings.FIRST_SUPERUSER,
-            first_superuser_password=settings.FIRST_SUPERUSER_PASSWORD,
-            first_superuser_budget=5.55,
-            domain_name=settings.DOMAIN_NAME,
-        )
-    else:  # Uses the Owner root key to update node keys
-        setup = node.setup.first()
-        if setup.signing_key:
-            # the system has a key so use it
-            signing_key = SigningKey(
-                setup.signing_key.encode("utf-8"), encoder=HexEncoder
+        service_name = os.getenv("SERVICE_NAME", "")
```

Comment:
MAYBE : Since we create the initial user in `prestart.sh` through `initial_data.py` , we could removed this line execution from prestart.sh, and in the docker compose in the backend service, we could add it to the command like
`command: /bin/sh -c " ./start.sh && python /app/grid/initial_data.py"`

Reply:
I think it might be `/bin/sh or /bin/ash`

Reply:
Thoughts?

Reply:
I imagine this would be the ideal design, wrapping and isolating the launch functionality into the backend build.

However, this method does more than just initialize a new configuration; it also reloads updated database information such as node credentials (root signature keys). So we also need to run it in backend-stream and celery worker containers, so that the keys between the three images are the same.

Reply:
What I did though, was split these two features (create and load ) into two different methods (`init_db` and `load_db`) and then delegated the responsibility of knowing which one of these we should execute to the `init_db.py` file.

Reply:
Great find, I initially thought , it was like a common node object , since the containers are isolated, it makes sense that node objects are new instances.